packages/login/src/components/LoginForm/LoginForm.js

```diff
@@ -26,6 +26,8 @@ export class LoginForm extends Component {
   }
 
   render() {
+    this.passwordFocus = !!this.props.username
```

Comment:
should be a local variable. check if prop has been set on the correct component in the test.